tests/data/greek-verb-paradigm.test.js

```diff
@@ -0,0 +1,856 @@
+
+  it('2 - checked Verb Paradigm2 - βουλευέσθων', async () => {
+    const inflectionsViewSet = await getInflectionSet('βουλευέσθων', Constants.LANG_GREEK)
+
+    expect(inflectionsViewSet.hasMatchingViews).toBeTruthy()
+    
+    expect(inflectionsViewSet.matchingViews.length).toEqual(1)
+    expect(inflectionsViewSet.matchingViews[0].constructor.name).toEqual('GreekVerbParadigmView')
+    expect(inflectionsViewSet.matchingViews[0].title).toEqual('ω-Verbs: Present System Middle-Passive')
+    expect(inflectionsViewSet.matchingViews[0].paradigm).toBeDefined()
+    expect(inflectionsViewSet.matchingViews[0].paradigm.paradigmID).toEqual('verbpdgm2')
+    expect(inflectionsViewSet.matchingViews[0].paradigm.title).toEqual('ω-Verbs: Present System Middle-Passive')
+    expect(inflectionsViewSet.matchingViews[0].linkedViews.length).toEqual(0)
+  })
+
+  it('3 - no matches - checked Verb Paradigm3', async () => {
+   
+  })
+
+  it('4, 8, 9 - checked Verb Paradigm4, 8, 9 - βουλεύσω', async () => {
+    const inflectionsViewSet = await getInflectionSet('βουλεύσω', Constants.LANG_GREEK)
+
+    expect(inflectionsViewSet.hasMatchingViews).toBeTruthy()
+    expect(inflectionsViewSet.matchingViews.length).toEqual(3)
+
+    expect(inflectionsViewSet.matchingViews[0].constructor.name).toEqual('GreekVerbParadigmView')
+    expect(inflectionsViewSet.matchingViews[0].title).toEqual('Weak (1st) Aorist System Active')
+    expect(inflectionsViewSet.matchingViews[0].paradigm).toBeDefined()
+    expect(inflectionsViewSet.matchingViews[0].paradigm.paradigmID).toEqual('verbpdgm8')
+    expect(inflectionsViewSet.matchingViews[0].paradigm.title).toEqual('Weak (1st) Aorist System Active')
+    expect(inflectionsViewSet.matchingViews[0].linkedViews.length).toEqual(0)
+
+    expect(inflectionsViewSet.matchingViews[1].constructor.name).toEqual('GreekVerbParadigmView')
+    expect(inflectionsViewSet.matchingViews[1].title).toEqual('Future System (Active and Middle) with contraction in -έω')
+    expect(inflectionsViewSet.matchingViews[1].paradigm).toBeDefined()
+    expect(inflectionsViewSet.matchingViews[1].paradigm.paradigmID).toEqual('verbpdgm4')
+    expect(inflectionsViewSet.matchingViews[1].paradigm.title).toEqual('Future System (Active and Middle) with contraction in -έω')
+    expect(inflectionsViewSet.matchingViews[1].linkedViews.length).toEqual(0)
+
+    expect(inflectionsViewSet.matchingViews[2].constructor.name).toEqual('GreekVerbParadigmView')
+    expect(inflectionsViewSet.matchingViews[2].title).toEqual('Weak (1st) Aorist System Middle')
+    expect(inflectionsViewSet.matchingViews[2].paradigm).toBeDefined()
+    expect(inflectionsViewSet.matchingViews[2].paradigm.paradigmID).toEqual('verbpdgm9')
+    expect(inflectionsViewSet.matchingViews[2].paradigm.title).toEqual('Weak (1st) Aorist System Middle')
+    expect(inflectionsViewSet.matchingViews[2].linkedViews.length).toEqual(0)
+  })
+
+  it('5 - no matchesx - checked Verb Paradigm5', async () => {
+  })

```

Comment:
same as prior question - does this table not work?

Reply:
I am not sure that it doesn't work, but I was not able to find the word to get it.

Reply:
@monzug have you found any words that (correctly) produce paradigm5?

Reply:
nope, because of the dialets. I believe that if we remove the dialet, the match will work

Reply:
ok, let's add these tests when that issue is fixed then.